app/views/users/show.html.erb

```diff
@@ -1,5 +1,32 @@
-<h2>User Details</h2>
-<p>Name: <%= @user.name %></p>
-<p>Email: <%= @user.email %></p>
-<p>Bio: <%= @user.bio %></p>
-<p>Total Posts: <%= @user.posts_counter %></p>
+<div class="user-profile-container">
+  <div class="user-profile">
+    <div class="user-image">
+      <%= image_tag @user.photo, alt: @user.name, class: "profile-photo" %>
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+      <div class="user-bio">
+        <h3>Bio</h3>
+        <p><%= @user.bio %></p>
+      </div>
+    </div>
+  </div>
+
+  <div class="user-posts">
+    <h3>Posts:</h3>
+    <% @user.posts.take(3).each do |post| %>
+      <div class="user-post">
+        <h4><%= link_to post.title, user_post_path(@user, post) %></h4>
+        <p><%= post.body.truncate(100) %></p>
+        <div class="post-stats">
+          <span>Comments: <%= post.comments_counter %></span>
+          <span>Likes: <%= post.likes_counter %></span>
+        </div>
+      </div>
+    <% end %>
```

Comment:
- [ ] As mentioned by the previous reviewer when we are visiting `users/:id` page there should be 3 most recent posts by the user on display and you can easily do that by making use of the method in your user model `recent_posts`. Example snippet:

```rb

# app/views/users/show.html.erb

[...]

<% @user.recent_post.each do |post| %>

<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>

<% end %>

<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>


```

WireFrame| your App
--|--
<img width="445" alt="Screenshot 2023-10-13 at 7 29 37 PM" src="https://github.com/cancelei/blog-in-rails/assets/44798044/18d36a77-49d1-4938-bc64-582b1f7a0f0b">|<img width="779" alt="Screenshot 2023-10-13 at 7 19 23 PM" src="https://github.com/cancelei/blog-in-rails/assets/44798044/46f0b75a-c7ea-495d-84b1-4c06e2b281e9">